deploy-chart.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+NEXUS_USERNAME="${NEXUS_USERNAME:?NEXUS_USERNAME not specified or empty}"
+NEXUS_PASSWORD="${NEXUS_PASSWORD:?NEXUS_PASSWORD not specified or empty}"
+CHARTS_DIR="${1:-charts}"
+
+CHART_DIR="${CHARTS_DIR}/spinnaker"
+TMP_DIR="$(mktemp -d)"
+
+trap 'rm -rf -- "${TMP_DIR}"' INT TERM HUP EXIT
+helm3 package --dependency-update "${CHART_DIR}" --destination "${TMP_DIR}"
```

Comment:
The `--dependency-update` flag ensures all dependencies from the `requirements.lock` are resolved.